dataprofiler/profilers/graph_profiler.py

```diff
@@ -399,9 +401,21 @@ def _get_continuous_distribution(
                 best_mle: float = 1000
                 best_fit_properties: tuple = None  # type: ignore[assignment]
 
+                scipy_updated = False
+                scipy_version = version.parse(importlib.metadata.version("scipy"))
+                if scipy_version >= version.parse("1.11.0"):
+                    scipy_updated = True
```

Comment:
Let's do two things here:
1. move it outside the for loop above as well since we don't need to recalc in the loop.
    * if we really want to we could also do it towards the top of the file.
2. instead of `scipy_updated` let's clarify the variable. Ideally the code reads as natural language so `scipy_updated` is quote ambiguous. I suggest `scipy_gte_1_11_0`